### ISSUE TITLE
Cannot compile m112

### ISSUE THREAD

**Issue by leo150** [VLprojects/mediasoup-client-swift#11]
I'm getting errors while trying to build the project. Looks like cmake tries to build libmediasoupclient using C++14 instead of C++17. Any idea what could be wrong? Thanks.
```
-- Build files have been written to: /User/myuser/mediasoup-client-swift/build/libmediasoupclient/device/arm64
[  4%] Building CXX object libmediasoupclient/libsdptransform/CMakeFiles/sdptransform.dir/src/grammar.cpp.o
[  8%] Building CXX object libmediasoupclient/libsdptransform/CMakeFiles/sdptransform.dir/src/parser.cpp.o
[ 12%] Building CXX object libmediasoupclient/libsdptransform/CMakeFiles/sdptransform.dir/src/writer.cpp.o
[ 16%] Linking CXX static library libsdptransform.a
[ 16%] Built target sdptransform
[ 20%] Building CXX object libmediasoupclient/CMakeFiles/mediasoupclient.dir/src/Consumer.cpp.o
In file included from /User/myuser/mediasoup-client-swift/Mediasoup/dependencies/libmediasoupclient/src/Consumer.cpp:3:
In file included from /User/myuser/mediasoup-client-swift/Mediasoup/dependencies/libmediasoupclient/include/Consumer.hpp:5:
In file included from /User/myuser/mediasoup-client-swift/Mediasoup/dependencies/webrtc/src/api/media_stream_interface.h:27:
In file included from /User/myuser/mediasoup-client-swift/Mediasoup/dependencies/webrtc/src/api/video/recordable_encoded_frame.h:16:
In file included from /User/myuser/mediasoup-client-swift/Mediasoup/dependencies/webrtc/src/api/units/timestamp.h:21:
In file included from /User/myuser/mediasoup-client-swift/Mediasoup/dependencies/webrtc/src/api/units/time_delta.h:22:
/User/myuser/mediasoup-client-swift/Mediasoup/dependencies/webrtc/src/rtc_base/units/unit_base.h:256:44: error: no template named 'is_floating_point_v' in namespace 'std'; did you mean 'is_floating_point'?
            typename std::enable_if_t<std::is_floating_point_v<T>>* = nullptr>
                                      ~~~~~^~~~~~~~~~~~~~~~~~~
                                           is_floating_point
```

**Comment by leo150** [comment]
Looks like `build.sh` is [pointing](https://github.com/VLprojects/mediasoup-client-swift/blob/ffbb6f4641db97b89a349a096baaf8163e763fd7/build.sh#L86) to a non-existing branch. Changing it to v3 fixed the issue.

**Comment by fedulvtubudul** [comment]
> Looks like `build.sh` is [pointing](https://github.com/VLprojects/mediasoup-client-swift/blob/ffbb6f4641db97b89a349a096baaf8163e763fd7/build.sh#L86) to a non-existing branch. Changing it to v3 fixed the issue.

Thanks for pointing this out. [vl-m112.0](https://github.com/VLprojects/libmediasoupclient/releases/tag/vl-m112.0) and [vl-3.4.0.0](https://github.com/VLprojects/libmediasoupclient/releases/tag/vl-3.4.0.0) are tags, not branches on our fork of libmediasoupclient repo. `v3` is a branch currently looking onto [vl-m112.0](https://github.com/VLprojects/libmediasoupclient/releases/tag/vl-m112.0), so I'll fix the build script next time I get my hands on it.

**Comment by fedulvtubudul** [comment]
Fixed in version 0.4.1
